### PR TITLE
docs about powdr as library

### PR DESCRIPTION
**Pull request by leonardoalt**

### CODE REVIEW

File: book/src/powdr_crate.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+```toml
+[dependencies]
+powdr = { git = "https://github.com/powdr-labs/powdr", branch = "main" }
+```
+
+The following Rust code has the same workflow as our previous "Hello World" example.
+The full project can be found [here](https://github.com/powdr-labs/powdr-hello-world).
+
+```rust
```
Comment (georgwiese) [inline book/src/powdr_crate.md:18]: Would there be a way to test this file? How about adding it as an example to the crate?

Comment (leonardoalt) [reply book/src/powdr_crate.md:18]: Yea maybe I can actually add this as a test, and then include it from the file into the docs. I think some asm files are like this

**Review by georgwiese — COMMENTED** [review]
I tried adding this as an example and it works (just have to change the path to the asm file). Think it would be good to have an example anyway and then we could add a CI job to run it!

File: book/src/powdr_crate.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Add `powdr` to your crate's dependencies:
+
+```toml
+[dependencies]
+powdr = { git = "https://github.com/powdr-labs/powdr", branch = "main" }
+```
+
+The following Rust code has the same workflow as our previous "Hello World"
+example.  The full project can be found
+[here](https://github.com/powdr-labs/powdr-hello-world), and as an example in
```
Comment (georgwiese) [inline book/src/powdr_crate.md:17]: Is there a difference between this and `powdr/examples/powdr_crate_usage.rs`?

File: book/src/powdr_crate.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+the powdr crate. To run the example in the [powdr
+repository](https://github.com/powdr-labs/powdr), run:
+
+```bash
+cargo run --example powdr_crate_usage
```
Comment (georgwiese) [inline book/src/powdr_crate.md:22]: Should be just add this Line to the CI config?

File: book/src/powdr_crate.md (hunk 2 above)
Comment (leonardoalt) [reply book/src/powdr_crate.md:17]: no but I wanted to give an example with a full crate also with the import, so people can potentially fork it and go from there

File: book/src/powdr_crate.md (hunk 3 above)
Comment (leonardoalt) [reply book/src/powdr_crate.md:22]: we could actually add a job for all "examples"

File: Cargo.toml [hunk 4]
```diff
@@ -4,6 +4,7 @@ resolver = "2"
 
 members = [
     "powdr",
+	"powdr-test",
```
Comment (georgwiese) [inline Cargo.toml:7]:
```suggestion
    "powdr-test",
```

**Review by georgwiese — APPROVED** [review]
LGTM! Just a tab-vs-spaces thing...
Also, you could squash the commits.

File: Cargo.toml (hunk 4 above)
Comment (leonardoalt) [reply Cargo.toml]: done, both

**Review by georgwiese — APPROVED** [review]

**Pull request merged**